mailboxes/boxes-lib.pl

```diff
@@ -965,7 +965,7 @@ sub send_mail
 						'pass' => $pass } );
 		&error("Failed to create Authen::SASL object") if (!$sasl);
 		local $conn = $sasl->client_new("smtp", &get_system_hostname());
-		local $arv = &smtp_command($h, "auth $auth\r\n", 1);
+		local $arv = &smtp_command($h, uc "auth $auth\r\n", 1);
```

Comment:
Better to write this as : 

```
local $arv = &smtp_command($h, "AUTH ".uc($auth)."\r\n", 1);
```

Reply:
Thanks for the feedback. Changed in the next commit.